less/core/tooltip.less

```diff
@@ -0,0 +1,21 @@
+.tooltip {
+  display: none;
+  position: absolute;
```

Comment:
For fixed positioned tooltips this should be position:fixed and the top, left won't have a scroll offset applied.

You can figure out if the $mouseoverEl is fixed position using something like this:
```js
const isFixedPosition = Boolean($mouseoverEl.parents().add($mouseoverEl).filter((index, el) => $(el).css('position') === 'fixed').length);
```

Reply:
@oliverfoster if I apply position:fixed to a test element and mouseover it the position of the tooltip is correct. could you provide a quick example of why we need to make the suggested change?

Reply:
Ah I see - scrolling while the tooltip it is visible - thanks!